tests/NLog.UnitTests/Layouts/LayoutTypedTests.cs

```diff
@@ -42,11 +44,11 @@ public class LayoutTypedTests : NLogTestBase
         [Fact]
         public void LayoutFixedIntValueTest()
         {
-            // Arrange
+            // Arrange            
```

Comment:
Possible that you can remove these extra white-spaces?

Reply:
Done.

I've noticed file headers retain trailing spaces, so, I turn off an extension (trailing white spaces) in Visual Studio when working on NLog.

What would be the correct way to handle the situation?

Reply:
Yes I have notice that other developers has extensions included that does different bonus formatting on save-file.

Guess one could "optimize" the NLog-repo to be more friendly towards bonus-formatting-developers.